pyomo/contrib/incidence_analysis/interface.py

```diff
@@ -16,12 +16,17 @@
 from pyomo.core.base.reference import Reference
 from pyomo.core.expr.visitor import identify_variables
 from pyomo.common.collections import ComponentSet, ComponentMap
-from pyomo.common.dependencies import scipy_available
+from pyomo.common.dependencies import scipy_available, networkx_available
 from pyomo.contrib.incidence_analysis.matching import maximum_matching
 from pyomo.contrib.incidence_analysis.triangularize import block_triangularize
+from pyomo.contrib.incidence_analysis.dulmage_mendelsohn import (
+        dulmage_mendelsohn,
+        )
```

Comment:
PEP8: your editor appears to indent this 98 spaces and not 4

Reply:
Seconding this comment.